### DISCUSSION TITLE
Error: 8: Failed to execute 'insertBefore' on 'Node': The node before which the new node is to be inserted is not a child of this node.

### DISCUSSION THREAD

**Discussion by Esteban-ricardo** [AntlerVC/firetable#944] [Support / Q&A] [answered]
👉 **Please describe how to reproduce this bug here.**

**Comment by notsidney** [comment, marked as answer]
Hi @Esteban-ricardo, please provide information about how you reached this issue.

What page were you on? What buttons did you click? What were you trying to do?

**Reply by BeeBombshell** [reply]
Hey guys, closing this discussion since it has been unattended for a long time. @Esteban-ricardo if you're still facing this issue, feel free to create a new one.
